gadget/install/partition_test.go

```diff
@@ -432,11 +433,11 @@ func (s *partitionTestSuite) TestRemovePartitions(c *C) {
 				},
 
 				StartOffset: 1024*1024 + 1024*1024,
-				Index:       2,
+				YamlIndex:   gadget.UnknownLaidOutDiskIndex,
```

Comment:
the name of the constant is strange now given DiskIndex existence and what the constant is used for/assigned to?

Reply:
do you think we still need to use/set the YamlIndex to this constant at all actually? it should hopefully be straight forward that this the YamlIndex is not to be used on an OnDiskStructure when there is DiskIndex

Reply:
it's probably OK to leave it out, I still hope one day to remove/clean up the messiness of OnDiskStructure vs LaidOutStructure etc

Reply:
ok, I removed this constant